Source/Core/VideoCommon/XFStructs.cpp

```diff
@@ -44,7 +37,7 @@ static void XFRegWritten(int transferSize, u32 baseAddress, DataReader src)
     case XFMEM_STATE1:  // internal state 1
     case XFMEM_CLOCK:
     case XFMEM_SETGPMETRIC:
-      nextAddress = 0x1007;
+      // Not implemented
```

Comment:
What isn't implemented?

Reply:
All of the above XF commands (`XFMEM_ERROR` through `XFMEM_SETGPMETRIC`).  They weren't implemented before, but I added the comment here to make that clear (and also I got rid of the system where it would skip over multiple commands since it was unnecessarily complicated for no real benefit.)

Reply:
Then, the comment makes more sense above the commands.